content/zh/_index.html

```diff
@@ -0,0 +1,196 @@
++++
+title = "CUE"
+linkTitle = "CUE"
++++
+
+
+{{< blocks/cover title="" image_anchor="top" height="min" color="light"  >}}
+<div class="col-lg-6">
+	<h3>
+		<img alt="CUE" width="130" src="/images/cue.svg" />
+		配置 <span style="font-size: 0.7em">(<b>C</b>onfigure)</span>
+		合并 <span style="font-size: 0.7em">(<b>U</b>nify)</span>
+		执行 <span style="font-size: 0.7em">(<b>E</b>xecute)</span>
+	</h3>
+	<p class="display-2 mt-3 mb-0">
+		验证、定义、使用动态以及文本式的数据
+	</p>
+	<div class="pt-3 mt-3 lead text-center">
+		<div class="mx-auto">
+			<a class="btn btn-lg btn-primary mr-3 mb-4" href="/zh-cn/docs" >
+			了解更多 <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+			</a>
+			<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://cuelang.org/releases">
+				下载 <i class="fab fa-github ml-2 "></i>
+			</a>
+		</div>
+	</div>
+</div>
+<div class="col-lg-6 mt-5 mt-lg-3 d-sm-block" style="display: none">
+	<asciinema-player
+			loop autoplay speed=2 font-size="12px" rows=17 cols=80
+			src="/cue.cast"></asciinema-player>
+</div>
+<div class="col-12 pt-3">
+	<div class="pt-3 lead text-center">
+		<div class="mx-auto">
+			<p class="lead mt-3">"掌控你的数据"</p>
+			<div class="mx-auto mt-3">
+				{{< blocks/link-down color="light-secondary" >}}
+			</div>
+		</div>
+	</div>
+</div>
+{{< /blocks/cover >}}
+
+
+{{% blocks/lead color="secondary" %}}
+**CUE** 是一门开源语言，有丰富的 API 和 工具链。可以用于定义、生成以及校验各种各样的数据：
+
+配置、API、数据库结构、代码等等，应有尽有。
+{{% /blocks/lead %}}
+
+
+{{< blocks/section color="dark" >}}
+
+<!-- todo: replace with zh-cn urls-->
+{{% blocks/feature icon="fa-check-circle" title="数据验证" url="/docs/usecases/validation/" %}}
+校验文字式的数据文件或程序数据，
+比如传入的 RPC 数据或数据库文档文件。
+{{% /blocks/feature %}}
+
+<!-- todo: replace with zh-cn urls-->
+{{% blocks/feature icon="fa-list-alt" title="配置" url="/docs/usecases/configuration/" %}}
+能对已有的数据（CUE、YAML、JSON）添加校验，同时也能减少大规模配置中的样板代码。
+{{% /blocks/feature %}}
+
+
+<!--
+life-ring map newspaper pause-circle play-circle stop-circle user clipboard comments (two bubbles) file-alt file-code handshake map -->
+<!-- todo: replace with zh-cn urls-->
+{{% blocks/feature icon="fa-handshake" title="结构定义"  url="/docs/usecases/datadef/" %}}
+定义 API 或某个标准通信的结构并且验证向后兼容性。
+<br>
+<br>
+{{% /blocks/feature %}}
+
+<!-- todo: replace with zh-cn urls-->
+{{% blocks/feature icon="fa-file-code" title="生成代码和结构" url="/docs/usecases/generate/" %}}
+
+使用一份代码校验并生成不同代码类型，比如 Protobuf 的定义和 OpenAPI 的定义。
```

Comment:
'不管在代码库、Protobuf 定义还是 OpenAPI 定义中，都能保持验证代码一致。'
Will it be better here? I know `keep ... in sync` should be translated to `保持同步`, 
but if you mean the validation code is same or consistent across different sources?

Reply:
> but if you mean the validation code is same or consistent across different sources?

Correct. One of the key advantages of CUE is that you can define a single source of truth for schemas, validation whatever it might be. That single source of truth might be CUE itself, or a different encoding. But the idea (or vision in the case of the encodings we don't fully support right now) is that CUE is the glue between different encodings, a glue that enables you to translate from your source of truth to different encodings so that the same schema, validation etc can be consistently used in different contexts. 

A good example might be an API between frontend and backend, where there is also some validation defined as part of the API. The backend is of course responsible for ultimately enforcing the validation, but it might be that in a frontend you want to drive UI logic using the same validation logic so that the user can enjoy more immediate validation feedback. In this context, regardless of the language of specification of the API and validation, CUE would enable you to translate into whatever representation you need for the backend (Go code example) _and_ the frontend (TypeScript perhaps). No human involvement required to redeclare/implement the validation in either the backend or frontend. Hence all parts of the system are "in sync".

Reply:
Deeply appreciate your DETAILED and IMPECCABLE explanation to it. I'm really flattered! 🙈
I have better understanding about it now. Thanks again!

And I've tweaked it a little bit, make it fluent and easy to understand. Please take a look.